src/runtime/server.ts

```diff
@@ -77,8 +80,44 @@ export async function createServer(
     });
   }
 
-  const routes = await getFunctionsAndAssets(config.baseDir);
-  const routeMap = setRoutes(routes);
+  let routes = await getFunctionsAndAssets(config.baseDir);
+  let routeMap = setRoutes(routes);
+
+  if (config.live) {
+    let seenFiles = new Set();
+
+    watch(config.baseDir, { recursive: true }, async (evt, name) => {
+      let relativeName = path.relative(config.baseDir, name);
+
+      if (!relativeName.match(/^(assets|functions)\//)) {
```

Comment:
If you could check out the library mentioned above, it seems like it supports creating a glob pattern. So we could be even more precise. Something like:
```
chokidar.watch(`${config.baseDir}/{functions/**/*.js,assets/**/*}`)
  .on('add', ...)
  .on('remove', ...)
```

Reply:
I've done this.  I used instead:

```
    const watcher = chokidar.watch(
      [
        path.join(config.baseDir, '/functions/**/*.js'),
        path.join(config.baseDir, '/assets/'),
      ],
...
    )
```